pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -388,6 +397,75 @@ class OSRow extends React.Component {
+    const validationStateUserPassword = validationFailed.userPassword ? 'error' : undefined;
+    let unattendedInstallationCheckbox = (
+        <label className="checkbox-inline">
+            <input id="unattended-installation" type="checkbox"
+                checked={unattendedInstallation}
+                disabled={unattendedDisabled}
+                onChange={e => onValueChanged('unattendedInstallation', e.target.checked)} />
+            {_("Run unattended installation")}
+            <FieldLevelHelp content={_("Perform an unattended install using libosinfo's install script support")} close />
+        </label>
+    );
+    if (unattendedDisabled) {
+        unattendedInstallationCheckbox = (
+            <OverlayTrigger overlay={ <Tooltip id='os-unattended-installation-tooltip'>{ _("The selected Operating System does not support unattended installations") }</Tooltip> } placement='left'>
+                {unattendedInstallationCheckbox}
+            </OverlayTrigger>
+        );
+    }
+
+    return (
+        <>
+            {unattendedInstallationCheckbox}
+            {!unattendedDisabled && unattendedInstallation ? <>
+                {os.profiles.length > 0 && <>
+                    <label className="control-label" htmlFor="profile-select">
+                        {_("Profile")}
+                    </label>
+                    <Select.Select id="profile-select"
+                        initial={os.profiles && os.profiles[0]}
+                        onChange={e => onValueChanged('profile', e)}>
+                        { (os.profiles || []).map(profile => {
+                            let profileName;
+                            if (profile == 'jeos')
+                                profileName = 'Server';
+                            else if (profile == 'desktop')
+                                profileName = 'Workstation';
+                            else
+                                profileName = profile;
+                            return <Select.SelectEntry data={profile} key={profile}>{profileName}</Select.SelectEntry>;
+                        }) }
+                    </Select.Select>
+                </>}
+                <label htmlFor='root-password' className='control-label'>
+                    {_("Root Password")}
+                </label>
+                <FormGroup validationState={validationStateRootPassword} bsClass='form-group ct-validation-wrapper' controlId='root-password'>
+                    <Password id='root-password' onValueChanged={(value) => onValueChanged('rootPassword', value)} />
+                    <HelpBlock>
+                        <p className="text-danger">{validationFailed.rootPassword}</p>
+                    </HelpBlock>
+                </FormGroup>
+                <label htmlFor='user-password' className='control-label'>
+                    {_("User Password")}
+                    <FieldLevelHelp content={_("The username is your current host username")} close />
```

Comment:
Does it make sense to have this infoblock which informs user why they cannot set username and what the username will be next to password field? If I would be user wondering why I can't set the username, or what the username will be, I wouldn't look for the answer in the password's helpblock :D. Perhaps let's not use tooltip, but instead put the information in a simple sentence above` User Password` row? @garrett WDTY?
![Screenshot from 2019-09-30 17-43-13](https://user-images.githubusercontent.com/42733240/65894295-cf652000-e3a9-11e9-94ea-391fccff36b9.png)

Reply:
Yeah, this part is tricky, especially since there are _two_ password fields. 

Usually, when there are two password fields, the second is a confirmation. That's not the case here... but I think a lot of people will treat it like that pattern. (Just like the checkbox on our log in page is not for remembering the password, but for having admin rights. That throws some people off too. :unamused:)

We could change it to something like:

"_Password for_ $USER" where $USER is the username.

If they don't want to have their own user account on the system, do they just leave the password blank? If so, we could change the popup text to make it a bit more clear. It then could be something like, "_Your current account will be set up on the virtual machine. Leave the password blank if you do not wish to have a user account created_."

Reply:
Not all distribution supports that though - for example debian will not allow you to now set a user password and only proceed with root.

Reply:
Fixed.

Reply:
I put the text bellow the password line, do you really prefer to make it a pop up?